assets/icons/wordpress-icons.js

```diff
@@ -9,31 +9,40 @@ import { Path, SVG } from '@wordpress/components';
  */
 export const chevronRight = (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M10.6 6L9.4 7l4.6 5-4.6 5 1.2 1 5.4-6z" />
+		<Path fill="currentColor" d="M10.6 6L9.4 7l4.6 5-4.6 5 1.2 1 5.4-6z" />
```

Comment:
This is usually controlled via CSS, and isn't added to the svg definitions in Gutenberg icons.

Reply:
Ah, reverted to match the original WP icons! =)
https://github.com/Automattic/sensei/pull/4205/commits/840101c42903c96437ddf219f5da32ed39028c9a

Also, the style is not needed anymore after we refactored to use the `<Button />`.